changes/5635.housekeeping

```diff
@@ -0,0 +1 @@
+Added borders to tabbed sections of mkdocs.
```

Comment:
Possibly change to a `.documentation` rather than a `.housekeeping`? Not critical.